setup.py

```diff
@@ -48,7 +47,7 @@
 
 install_req = ['scipy>=0.15',
                'matplotlib>=1.2, !=2.1.0, !=2.1.1',
-               'numpy>=1.10, !=1.13.0',
+               'numpy>=1.11, !=1.13.0',
```

Comment:
Why is this needed? It's ``dask`` who requires v1.11 not HyperSpy.

Reply:
Yes, indeed!